docs/package.json

```diff
@@ -82,6 +82,9 @@
     "express": "^4.17.1",
     "gatsby-plugin-express": "^1.1.6",
     "gatsby-plugin-react-helmet": "^3.3.10",
+    "gatsby-remark-autolink-headers": "^2.3.13",
+    "gatsby-remark-table-of-contents": "^0.1.1",
+    "gatsby-transformer-remark": "^2.8.35",
     "helmet": "^4.1.0",
```

Comment:
i'm not seeing this packages being used in the gatsby config

Reply:
agh, they aren't.  relics of trying other things